M2/Macaulay2/packages/PositivityToricBundles.m2

```diff
@@ -486,8 +482,7 @@ if separatesJets(tvb, Verbosity=>opts#Verbosity) >= 0 then true else false;
 --           check if the vector bundle is very ample, using [RJS, Cor. 6.7]
 --   INPUT : 'tvb', toric vector bundle
 --  OUTPUT :  'true' if very ample, otherwise 'false'
---isVeryAmple = method( Options => true ) -- conflicts with Polyhedra
---isVeryAmple (ToricVectorBundleKlyachko) := (tvb) -> if separatesJets(tvb) >= 1 then true else false;
+--isVeryAmple = method( Options => true ) -- already defined in Polyhedra
 isVeryAmple (ToricVectorBundleKlyachko) := {Verbosity => 0} >> opts -> tvb -> if separatesJets(tvb, Verbosity=>opts#Verbosity) >= 1 then true else false
```

Comment:
This is fine, but just so you know, this also works:
```suggestion
isVeryAmple (ToricVectorBundleKlyachko) := {Verbosity => 0} >> opts -> tvb -> separatesJets(tvb, opts) >= 1
```

Also, it might be better to check the global variable `debugLevel` for printing extra information rather than adding a `Verbosity` option.

Reply:
Thanks for the hint! I will think about using `debugLevel` and probably implement it (hopefully soon).

Reply:
May I ask a question about using `debugLevel`? 
One reason for using the `Verbosity` option is that many methods (more complicated than the `isVeryAmple` above) pass `Verbosity-1` to invoked other methods. So the value of `Verbosity` decides how far you want to follow into the rabbit hole of nested methods. 
It seems to me that this behaviour is not possible with `debugLevel` alone, or is there some counter of the level of depth? It seems to me that `interpreterDepth` or `recursionDepth()` count something else...

Reply:
Well, in a high level method you can check `if debugLevel > 0`, on nested methods you can check `if debugLevel > 1`, and on really deep ones you could even do something like `if debugLevel == 12` or something like that.

The benefit is that suppose someone wrote a method using `isVeryAmple` but didn't use the `Verbosity` option. Then there would be no way to get debugging info from it.

Reply:
Thanks for the answer! I guess, I will change at some point to use `debugLevel` (but not for this pull request).